src/styles/base/_lists.scss

```diff
@@ -15,6 +15,13 @@ ol {
     margin-bottom: $small-spacing;
     padding-left: $base-spacing;
   }
+  &.form-list{
+    li{
```

Comment:
This should be on a separate component, also don't nest declarations if you can avoid it, putting a class works better